Kickstarter-iOS/Views/Cells/CommentCell.swift

```diff
@@ -109,6 +116,8 @@ final class CommentCell: UITableViewCell, ValueCell {
     self.flagButton.rac.hidden = self.viewModel.outputs.flagButtonIsHidden
     self.replyButton.rac.hidden = self.viewModel.outputs.replyButtonIsHidden
     self.viewRepliesStackView.rac.hidden = self.viewModel.outputs.viewRepliesStackViewIsHidden
+
+    self.postedButton.rac.hidden = self.viewModel.outputs.commentStatus.map { $0 != .retrySuccess }
```

Comment:
Did the formatter indent this? I would just alphabetize it again within this method for simplicity sake.

Reply:
This was suppose to be in the VM, I actually forgot it there and I didn't catch it in my review.